src/flows/components/header/SaveState.tsx

```diff
@@ -29,13 +29,14 @@ const SaveState: FC = () => {
     })
   }
 
-  if (
-    !isFlagEnabled('ephemeralNotebook') ||
-    isFlagEnabled('flowPublishLifecycle')
-  ) {
+  if (!isFlagEnabled('ephemeralNotebook')) {
     return null
   }
 
+  if (isFlagEnabled('flowPublishLifecycle')) {
```

Comment:
move this to being above ephemeral notebook

Reply:
k